### PR TITLE
Create listing and detail

### PR DESCRIPTION
**Pull request by landreaubis**

### CODE REVIEW

File: src/AppBundle/Controller/BlurayController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     * @Route("/Bluray/listing/{page}/{maxperpage}")
+     */
+    public function listingAction($page = 1, $maxperpage = 4)
+    {
+        $listBluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll();
+        $html = $this->container->get('templating')->render(
+            'bluray/listing.html.twig',
+            array('listBluray' => $listBluray)
+        );
+        return new Response($html);
```
Comment (Jean-Beru) [inline src/AppBundle/Controller/BlurayController.php:21]: You should use http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/view.html to symplify your code.

It's just a recommendation :wink:

File: src/AppBundle/Controller/BlurayController.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        $bluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->find($id);
+        if (!is_object($bluray)) {
+            throw $this->createNotFoundException();
+        }
+
+        $html = $this->container->get('templating')->render(
+                'bluray/detail.html.twig',
+                array('bluray' => $bluray)
+                );
+        return new Response($html);
```
Comment (Jean-Beru) [inline src/AppBundle/Controller/BlurayController.php:38]: Same as previous comment

**Comment by Jean-Beru** [comment]
:+1: 
But you can follow my recommendations if you want.

**Comment by landreaubis** [comment]
I use this doc : http://symfony.com/doc/current/book/page_creation.html it's for that.
Thanks for this recommandations.

I'll try it tonight

**Pull request closed without merging**